Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml

```diff
@@ -135,18 +112,10 @@ services:
         delay: 5s
         window: 10s
     env_file: 
-     - ./secrets/.grafana.env 
-    cap_drop:
-      - ALL
-    logging:
-      driver: "json-file"
-      options:
-        max-file: "3"
-        max-size: "5m"
-        tag: "{\"name\":\"{{.Name}}\",\"id\":\"{{.ID}}\"}"
+     - ./.grafana.env 
```

Comment:
Keep this env file in `secrets/` directory itself. Or update in the README secrets directory structure.